datastream-server-api/src/main/java/com/linkedin/datastream/server/DatastreamEventProducer.java

```diff
@@ -39,4 +39,12 @@
    * As such, it is not efficient to be invoked very frequently.
    */
   void flush();
+
+  /**
+   * Enable periodic flush on send
+   *
+   * @param enableFlushOnSend Whether to enable flushing on send or not
+   */
+  default void flushOnSend(boolean enableFlushOnSend) {
```

Comment:
Can you change it to enablePeriodicFlushOnSend as method name?

Reply:
done